packages/odyssey-react/src/components/TextInput/TextInput.test.tsx

```diff
@@ -153,5 +153,25 @@ describe("TextInput", () => {
     expect(ref).toHaveBeenLastCalledWith(screen.getByRole(textBox));
   });
 
+  it("renders the prefix", () => {
+    render(<TextInput label={label} prefix="test prefix" />);
+    const prefixElement = screen.getByText("test prefix");
+    expect(prefixElement).toBeInTheDocument();
+    expect(prefixElement.className).toEqual("prefix");
+  });
+
+  it("renders the suffix", () => {
+    render(<TextInput label={label} suffix="test suffix" />);
+    const suffixElement = screen.getByText("test suffix");
+    expect(suffixElement).toBeInTheDocument();
+    expect(suffixElement.className).toEqual("suffix");
+  });
```

Comment:
a simple test here for suffix/prefix for the focus handling behavior would be excellent to to help us avoid future a11y regressions!